amp-pwa-reader/gulpfile.js

```diff
@@ -131,15 +131,69 @@ function modularizeJS(name) {
   return "\nmodule.exports = " + name + ';';
 }
 
-var dist = gulp.series(gulp.parallel(copy, styles, scripts, server), inline, injectManifest);
+/* Generates a complete ServiceWorker, with precaching and runtime caching rules. */
+function buildSW() {
+    // This will return a Promise
+    return workboxBuild.generateSW({
+        globDirectory: './dist/',
+        // Static precaching of shell
+        globPatterns: [
+            'img/*.{svg,png,jpg}', 'index.html', 'inline.css'
+        ],
+        swDest: './dist/sw.js',
+        // Register main route for all navigation links to pages
+        navigateFallback: 'index.html',
+        navigateFallbackBlacklist: [/img\/.*/, /\.(js|css)/],
+        navigateFallbackWhitelist: [/./],
```

Comment:
That's the default, and you should be able to leave it out.

Reply:
Hi Jeff, according to the documentation, navigateFallback defaults to undefined (https://developers.google.com/web/tools/workbox/modules/workbox-build).
Is it safe, in this case, to remove these three lines, or only the ones for Blacklist/Whitelist?

Thanks!

Reply:
Sorry, I specifically meant that `navigateFallbackWhitelist` will default to `[/./]`, so you could potentially remove that one line. You should keep the other two lines.

(The default actually comes from https://github.com/GoogleChrome/workbox/blob/master/packages/workbox-routing/NavigationRoute.mjs#L56 rather than the build tool, which is... confusing.)

Reply:
Thank you Jeff